knowit/__main__.py

```diff
@@ -3,19 +3,22 @@
 
 import json
 import logging
-import os
 import sys
 from argparse import ArgumentParser
 
 from babelfish.language import Language
+
```

Comment:
The blank space it needed because a bug thus failing Travis, I don't like having extra space between the import is I ignored the error. But I should have removed the extra space again.